client/post-editor/editor-publish-button/index.jsx

```diff
@@ -92,13 +92,20 @@ export class EditorPublishButton extends Component {
 					return this.props.translate( 'Schedule…',
 						{ comment: 'Button label on the editor sidebar - a confirmation step will follow' } );
 				}
+
 				return this.props.translate( 'Schedule' );
 			case 'publish':
-				if ( this.props.isConfirmationSidebarEnabled ) {
-					return this.props.translate( 'Publish…',
-						{ context: 'Button label on the editor sidebar - a confirmation step will follow' } );
+				if ( ! this.props.isConfirmationSidebarEnabled ) {
+					return this.props.translate( 'Publish' );
 				}
-				return this.props.translate( 'Publish' );
+
+				if ( this.props.isPublishing ) {
+					return this.props.translate( 'Publishing…',
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Publishing...' )` *ES Score: 20*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove/editor-confirmation-sidebar-overlay)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).